src/main/java/uk/co/ramp/covid/simulation/place/Household.java

```diff
@@ -298,4 +267,35 @@ public void dayEnd() {
             isolationTimer--;
         }
     }
+
+    // Household Type management
+    protected int adults = 0;
+    protected int children = 0;
+    protected int pensioners = 0;
+
+    public abstract boolean adultRequired();
+    public abstract boolean adultAllowed();
+    public abstract boolean childRequired();
+    public abstract boolean childAllowed();
+    public abstract boolean pensionerRequired();
```

Comment:
I suggest adding default implementations for pensionerRequired() and pensionerAllowed()
```suggestion
    public boolean pensionerRequired() {
        return adultRequired();
    }
```

Reply:
I'm less keen to do this - while they are often similar (perhaps less so when we split the classes below), I think there's something to be said about forcing a compile error if we don't explicitly specify them. Else for something like SingleAdult we might forget to override pensionerRequired to false and silently get different behavior.

What we might do is use "adultRequired" as the method call within the subclass to reduce the duplication (explicitly saying for this subclass pensionerRequired is the same as adultRequired)?